plugins/woocommerce-admin/client/customize-store/intro/actions.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { assign, DoneInvokeEvent } from 'xstate';
+
+/**
+ * Internal dependencies
+ */
+import { customizeStoreStateMachineEvents } from '..';
+
+/**
+ * Internal dependencies
+ */
+import { customizeStoreStateMachineContext } from '../types';
+import { ThemeCard } from './theme-cards';
+
+export const assignThemeCards = assign<
+	customizeStoreStateMachineContext,
+	customizeStoreStateMachineEvents // this is actually the wrong type for the event but I still don't know how to type this properly
```

Comment:
It does seem a bit difficult to type this. 🤔

Reply:
I think xstate v5 has improvements to this situation 🤞